source/agora/consensus/state/ValidatorSet.d

```diff
@@ -616,9 +644,8 @@ public class ValidatorSet
             auto results = this.db.execute(
                 "SELECT preimage, enrolled_height, distance " ~
                 "FROM validator_set WHERE key = ? AND enrolled_height <= ? " ~
-                "AND enrolled_height + distance >= ? AND active = ?",
-                enroll_key, height.value, height.value,
-                EnrollmentStatus.Active);
+                "AND enrolled_height + distance >= ? ORDER BY active DESC",
```

Comment:
Was looking into this earlier, wouldn't the following work:
```suggestion
                "AND enrolled_height + distance >= ? ORDER BY enrolled_height + distance DESC",
```
?

Which we will later replace with `height` once we finish the `distance` => `height` change.

Reply:
Yes that would also work and prepare for the coming change. I'll update it.